src/algebra/monoid_algebra/basic.lean

```diff
@@ -336,13 +336,19 @@ subset.trans support_sum $ bUnion_mono $ assume a₁ _,
 
 section
 
+/-- Like `finsupp.map_domain_zero`, but for the `1` we define in this file -/
+@[simp] lemma map_domain_one {α : Type*} {β : Type*} {α₂ : Type*}
```

Comment:
```suggestion
@[simp] lemma map_domain_one {α β α₂ : Type*}
```
Is there a reason to not merge the Types in a single annotation?

Reply:
I'm just copying the style of the surrounding lemmas; but a cleanup of all of them in a future PR would probably be nice